tests/dummy/app/controllers/bars.js

```diff
@@ -22,7 +22,7 @@ export default Ember.Controller.extend({
   }),
 
   tableContent: Ember.computed(function() {
-    return _.range(100).map(function(index) {
+    return Array.apply(null, new Array(100)).map(function(_, index) {
```

Comment:
maybe a `for` loop would have been fine... is that what you mean? I was going for a drop-in substitution but now I'm think a for loop would have been better.